backend/app/domains/users/schemas.py

```
@@ -0,0 +1,60 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, EmailStr, ConfigDict
+
+from app.domains.students.schemas import StudentRead
+from app.domains.users.models import UserRole
+
+
+class UserCreate(BaseModel):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)